lua/entities/gmod_wire_expression2/core/strfunc.lua

```diff
@@ -43,34 +40,36 @@ local function findFunc( self, funcname, typeids, typeids_str )
 		if not func then
 			for i=#typeids,1,-1 do
 				func, func_return_type = checkFuncName( self, funcname .. "(" .. concat(typeids,"",1,i) .. "...)" )
-				if func then vararg = true break end
+				if func then break end
 			end
 
 			if not func then
 				func, func_return_type = checkFuncName( self, funcname .. "(...)" )
-				if func then vararg = true end
 			end
 		end
 
 		if not func then
 			for i=#typeids,2,-1 do
 				func, func_return_type = checkFuncName( self, funcname .. "(" .. typeids[1] .. ":" ..  concat(typeids,"",2,i) .. "...)" )
-				if func then vararg = true break end
+				if func then break end
 			end
 
 			if not func then
 				func, func_return_type = checkFuncName( self, funcname .. "(" .. typeids[1] .. ":...)" )
-				if func then vararg = true end
 			end
 		end
 	else
 		func, func_return_type = checkFuncName( self, funcname .. "()" )
 	end
 
-	if func then
-		local t = { str, func, func_return_type, vararg }
-		insert( self.strfunc_cache, 1, t )
-		if #self.strfunc_cache == 21 then self.strfunc_cache[21] = nil end
+	if func and not self.strfunc_cache[1][str] then
```

Comment:
Use tabs not spaces

Reply:
VSCode indent detection switched to spaces for some reason